### PR TITLE
#151025308 Modify the model methods

### PR DESCRIPTION
**Pull request by andela-oowonikoko**
#### What does this PR do?

- Modify the model methods

#### Description of Task to be completed?

- change the model methods to class methods from instance methods
- fix the code to reflect changes

#### How should this be manually tested?

At the root of the project directory run the following command:
```$pytest -v```

#### What are the relevant pivotal tracker stories?

[Delivers #151025308]

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -70,48 +70,55 @@ def delete(self):
...
         Query and filter the data of a model,
         returning the first result
         """
-        return self.query.filter_by(**kwargs).first()
+        return cls.query.filter_by(**kwargs).first()
 
+    @classmethod
     def filter_and_count(self, **kwargs):
         """Query, filter and counts all the data of a model"""
         return self.query.filter_by(**kwargs).count()
```
Comment (danielwangai) [inline api/models.py]: @andela-oowonikoko why are you using `self` instead of `cls` here?

Comment (andela-oowonikoko) [reply api/models.py]: Thanks for this, will fix it ASAP @danielwangai

**Comment by andela-uibrahim** [comment]
LGTM

**Comment by Andela-ddiei** [comment]
LGTM

**Pull request merged**